### DISCUSSION TITLE
Ability to split photo across 2 screens?

### DISCUSSION THREAD

**Discussion by grayhat917** [Geektoolkit/Dynaframe3#63] [Ideas]
Hello and happy new year.  Can you tell me if your app will allow for the splitting of a single image across 2 screns?  Ideally I'd like to build something like this:
![image](https://user-images.githubusercontent.com/31521838/103549541-8be34680-4e75-11eb-86f8-c418a0d0568b.png)

Thanks in advance for your guidance!

**Comment by qwksilver** [comment]
so one rpi3 or 4 will not manage multiple screens under dynaframe, this isn't dynaframe direcly both the picture and video viewer simply don't do it.

however.  you can resize/crop a photo to 2160 x 1920  and then crop that into left and right pieces that are 1080x1920. and run two dynaframes seperate hardware for each screen in multiframe sync mode.   

i can build you a fast process file so you can split them quick just by putting them in a directory and running a batch file on them if you want.  that will still require you to curate the base files (resize/crop a photo to 2160 x 1920).  but will then be able to cut them and separate them really fast.  there may be dynaframe internal functionality to do this coming, but won't be soon and will depend on demand.
